addons/point_of_sale/models/account_journal.py

```diff
@@ -7,19 +7,13 @@
 class AccountJournal(models.Model):
     _inherit = 'account.journal'
 
-    journal_user = fields.Boolean('Use in Point of Sale',
-        help="Check this box if this journal define a payment method that can be used in a point of sale.")
+    pos_payment_method_ids = fields.One2many('pos.payment.method', 'cash_journal_id', string='Point of Sale Payment Methods')
 
     @api.model
     def _search(self, args, offset=0, limit=None, order=None, count=False, access_rights_uid=None):
```

Comment:
I think we can remove this override of "_search", need to remove the context from here: https://github.com/odoo/odoo/blob/master/addons/point_of_sale/views/pos_order_view.xml#L9

Nothing else to change because it's handled in https://github.com/odoo/odoo/blob/master/addons/point_of_sale/wizard/pos_payment.py#L39